dashboard/app/helpers/script_levels_helper.rb

```diff
@@ -70,4 +70,51 @@ def tracking_pixel_url(script)
       CDO.code_org_url "/api/hour/begin_#{script.name}.png"
     end
   end
+
+  def hoc_finish_url(script)
+    if script.name == Script::HOC_2013_NAME
+      CDO.code_org_url '/api/hour/finish'
+    else
+      CDO.code_org_url "/api/hour/finish/#{script.name}"
+    end
+  end
+
+  def header_progress(script_level)
+    script = script_level.script
+    stage = script_level.stage
+
+    game_levels =
+      if current_user
+        current_user.levels_from_script(script, stage)
+      else
+        script.script_levels.to_a.select{|sl| sl.stage_id == script_level.stage_id}
```

Comment:
Line is too long. [83/80]<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.